inspirehep/dojson/utils/__init__.py

```diff
@@ -252,7 +252,11 @@ def split_page_artid(page_artid):
         if page_artid:
             if '-' in page_artid:
                 # if it has a dash it's a page range
-                page_start, page_end = page_artid.split('-')
+                page_range = page_artid.split('-')
+                if len(page_range) == 2:
+                    page_start, page_end = page_range
+                else:
+                    artid = page_artid
```

Comment:
I wonder what actual values there are on production. E.g. If it's 123- it's probably just a `start_page`. (-123 instead if of dubious meaning).